gandi/cli/commands/ip.py

```diff
@@ -77,32 +77,38 @@ def list(gandi, datacenter, type, id, attached, detached, version, reverse,
 
 
 @cli.command()
-@click.argument('resource')
+@click.argument('resources', nargs=-1)
 @pass_gandi
-def info(gandi, resource):
-    """Display information about an ip.
+def info(gandi, resources):
+    """Display information about one or more IPs
 
     Resource can be an ip or id.
     """
     output_keys = ['ip', 'state', 'dc', 'type', 'vm', 'reverse']
+    justify = 14
 
+    resources = sorted(tuple(set(resources)))
     datacenters = gandi.datacenter.list()
 
-    ip = gandi.ip.info(resource)
-    iface = gandi.iface.info(ip['iface_id'])
-    vms = None
-    if iface.get('vm_id'):
-        vm = gandi.iaas.info(iface['vm_id'])
-        vms = {vm['id']: vm}
-
-    output_ip(gandi, ip, datacenters, vms, {iface['id']: iface},
-              output_keys)
+    ret = []
+    for num, item in enumerate(resources):
+        if num:
+            gandi.separator_line()
+        ip = gandi.ip.info(item)
+        ret.append(ip)
+        iface = gandi.iface.info(ip['iface_id'])
+        vms = None
+        if iface.get('vm_id'):
+            vm = gandi.iaas.info(iface['vm_id'])
+            vms = {vm['id']: vm}
+        output_ip(gandi, ip, datacenters, vms, {iface['id']: iface},
+                  output_keys)
 
-    return ip
+    return ret
 
 
 @cli.command()
-@click.argument('ip')
+@click.argument('ip', nargs=-1)
```

Comment:
You should not use multiple args here in `update` command as we can only update one reverse per ip, I think that it makes no sense to update the same reverse for different ips (but I may be missing a use case here).